vaadin-rich-text-editor-flow-parent/vaadin-rich-text-editor-flow/src/main/java/com/vaadin/flow/component/richtexteditor/RichTextEditor.java

```diff
@@ -759,11 +803,11 @@ public RichTextEditorI18n setClean(String clean) {
         public String toString() {
             return "[" + undo + ", " + redo + ", " + bold + ", " + italic + ", "
                     + underline + ", " + strike + ", " + h1 + ", " + h2 + ", "
-                    + h3 + ", " + subscript + ", " + superscript + ", "
-                    + listOrdered + ", " + listBullet + ", " + alignLeft + ", "
-                    + alignCenter + ", " + alignRight + ", " + image + ", "
-                    + link + ", " + blockquote + ", " + codeBlock + ", " + clean
-                    + "]";
+                    + h3 + ", " + color + ", " + background + ", " + subscript
+                    + ", " + superscript + ", " + listOrdered + ", "
+                    + listBullet + ", " + alignLeft + ", " + alignCenter + ", "
+                    + alignRight + ", " + image + ", " + link + ", "
+                    + blockquote + ", " + codeBlock + ", " + clean + "]";
```

Comment:
@web-padawan `color` and `background` should be listed before `h1`. Apparently one of the tests (`initEditorWithCustomI18n_i18nUpdated`) relies on them to be in DOM order.